tutorials/Python/01_BasicTypes/BasicTypes.ipynb

```diff
@@ -108,7 +108,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "###### Exercise 1.1.1: "
+    "### Exercise 1.1.1: "
```

Comment:
Should sub-exercises be one more #?

I.e. ### for 1.1 and #### for 1.1.1 (indentation level depends on previous comment).

Reply:
See above, I think numbering is a bit off.

Reply:
Shall we leave it as it is for now and then when we go back through, make sure that it's more consistent?

My view is that we should probably just have own exercises be continuous, i.e. 1.X to 1.XX.
Thoughts?